dev-qt/prison/prison-3.41.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="A barcode api"
+DESCRIPTION="prison is a Qt-based barcode abstraction layer/library that \
+provides uniform access to generation of barcodes with data. The API supports \
+two types of barcodes: QRCode and DataMatrix."
+HOMEPAGE="https://api.kde.org/frameworks/prison/html/index.html"
+COPYRIGHT="2010-2017 KDE Foundations"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://download.kde.org/stable/frameworks/5.38/prison-5.41.0.tar.xz"
```

Comment:
The recipe should be renamed to prison-**5**.41.0.recipe or prison-5.39.0.recipe, but not prison-**3**.41.recipe
```diff
-SOURCE_URI="https://download.kde.org/stable/frameworks/5.38/prison-5.41.0.tar.xz"
+SOURCE_URI="https://download.kde.org/stable/frameworks/${portVersion%.*}/prison-$portVersion.tar.xz"
```